runtime/polkadot/src/lib.rs

```diff
@@ -578,7 +578,6 @@ impl pallet_staking::EraPayout<Balance> for EraPayout {
 
 		runtime_common::impls::era_payout(
 			total_staked,
-			// Polkadot has no notion of gilts, the entire issuance is non-guilt.
 			total_issuance,
```

Comment:
Shouldn't this also be "active balance", as of your new PR?

Reply:
IMO no, I considered this too. The `era_payout` should still consider balances in Crowdloans/Auctions/etc. as part of the total issuance, as they are part of the network, but people have chosen to use them for something outside of Staking. The Active balance is for the ability to be "active" with those funds at a given time (contributing to a crowdloan means you are willing to not be able to be active with them for the duration of the lease).